packages/messenger-widget/src/components/Contacts/bl.ts

```diff
@@ -195,13 +195,30 @@ const getMessagesFromUser = (
 
 export const setContactSelectedFromCache = (
```

Comment:
The function returns a number instead of a contract. So either rename it to setContactIndexSelectedFromCache or make it return a contact